actions.yaml

```diff
@@ -0,0 +1,8 @@
+restart-pg:
+  description: Restart the plumgrid-director unit's service.
+post-ips:
```

Comment:
As actions are available to the end user dont you think these names should be more descriptive? What if we add a prefix like sapi-post-ips so that the user does not confuse this action for something else.